components/molecules/client/user/component.tsx

```diff
@@ -0,0 +1,53 @@
+import FollowBtn from '@/components/molecules/client/follow-btn';
+import Image from 'next/image';
+
+interface UserProps {
+  userId: string;
+  postId: string;
+  initialFollowed: boolean;
+  name: string;
+  followersCount: number;
+  bio: string;
+  avatarUrl: string;
+}
+
+const UserComponent: React.FC<UserProps> = ({
+  userId,
+  name,
+  followersCount,
+  bio,
+  avatarUrl,
+}) => {
+  return (
+    <div className="flex items-center justify-between w-[460px] h-[89px] gap-md p-4 border border-light-neutral-gray rounded-md">
```

Comment:
careful with hard coding w/h. Just use "w-full h-fit"

Reply:
Also for padding, use the spacing values defined in tailwind.config.ts (make sure tailwindcss plugin is installed if you are using vscode)